example/android/app/src/main/java/com/example/ddsdkreactnative/MainApplication.java

```diff
@@ -34,7 +34,7 @@ public boolean getUseDeveloperSupport() {
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for DdSdkReactNativeExample:
           // packages.add(new MyReactNativePackage());
-          packages.add(new DdSdkReactNativePackage());
+          // packages.add(new DdSdkReactNativePackage());
```

Comment:
Where are those packages coming from ?

Reply:
Auto generated when you create a Native Module.